src/components/Admin/User/List/UserListTable.tsx

```diff
@@ -0,0 +1,92 @@
+import { ActionIcon, Avatar, Badge, Group, Table, Text } from '@mantine/core';
+import { IconArchive, IconArchiveOff, IconKey, IconTrash } from '@tabler/icons';
+import dayjs from 'dayjs';
+import { openUserPermissionModal } from '../UserPermissionModal';
+import { useUserListActions } from './actions';
+import { RouterOutputs } from '../../../../utils/api';
+
+type UserList = RouterOutputs['user']['list'];
+interface UserListTableProps {
+  users: UserList;
+}
+
+export const UserListTable = ({ users }: UserListTableProps) => (
+  <Table sx={{ minWidth: 800 }} verticalSpacing="sm">
+    <thead>
+      <tr>
+        <th>User</th>
+        <th>Role</th>
+        <th>Last active</th>
+        <th>Status</th>
+        <th>Actions</th>
+      </tr>
+    </thead>
+    <tbody>
+      {users.map((u) => (
```

Comment:
Can there be no users ? If the object is empty it would crash

Reply:
Why would it crash? ``map`` doesn't throw an exception, if the list is empty AFAIK